tools/cuda-version.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+nvcc --version | grep release | awk '{print $5}' | tr ',' ' '
```

Comment:
You need a newline here: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline